src/features/titles/settings.js

```diff
@@ -0,0 +1,180 @@
+				) }
+				{ ! paused && startTime >= Date.now() && (
+					<Notice>
+						{ __( 'Your test will start in' ) }
+						{ ' ' }
+						<Duration time={ startTime - Date.now() } />
+					</Notice>
+				) }
+				{ started && (
+					<CenteredButton
+						disabled={ titles.length < 2 }
+						isBusy={ isSaving }
+						onClick={ () => {
+							if ( arrayEquals( titles, prevTitles ) ) {
+								return setPaused( ! paused );
+							}
+
+							if ( paused && window.confirm( __( 'Are you sure? Editing the titles will reset the current test results.', 'altis-experiments' ) ) ) {
+								return setPaused( ! paused );
+							}
+
+							setPaused( ! paused );
+						} }
+					>
+						{ paused && __( 'Resume test', 'altis-experiments' ) }
+						{ ! paused && __( 'Pause test', 'altis-experiments' ) }
+					</CenteredButton>
+				) }
+				{ ! started && (
+					<CenteredButton
+						disabled={ titles.length < 2 }
+						isBusy={ isSaving }
+						onClick={ startTest }
+					>
+						{ __( 'Run test', 'altis-experiments' ) }
+					</CenteredButton>
+				) }
+				{ started && paused && ! arrayEquals( titles, prevTitles ) && (
+					<Fragment>
+						<Warning>
+							{ __( 'Editing the titles now will reset the test and delete the previous results.' ) }
+						</Warning>
+						<Button
+							isLink
+							onClick={ resetTitles }
+						>
+							{ __( 'Undo changes', 'altis-experiments' ) }
+						</Button>
+					</Fragment>
+				) }
+			</PanelRow>
+			<PanelRow>
+				<p>{ __( 'Add multiple post titles and see which one has a higher conversion rate.', 'altis-experiments' ) }</p>
+				<TitleTextField
+					defaultTitle={ title }
+					isEditable={ paused }
+					postId={ post.id }
+					onChange={ updateTitles }
+					titles={ titles }
+					variants={ variants }
+				/>
+			</PanelRow>
+			<PanelRow>
+				<TrafficPercentageField
+					value={ trafficPercentage || 35 }
+					onChange={ percent => updateTest( { traffic_percentage: percent } ) }
+				/>
+			</PanelRow>
+			<PanelRow>
+				<DateRangeField
+					startTime={ startTime || Date.now() }
+					endTime={ endTime || Date.now() + ( 30 * 24 * 60 * 60 * 1000 ) }
+					onChangeStart={ time => updateTest( { start_time: time } ) }
+					onChangeEnd={ time => updateTest( { end_time: time } ) }
+					description={ __( 'The test will stop automatically when it reaches statistical significance.', 'altis-experiments' ) }
+				/>
+			</PanelRow>
+			{ started && (
+				<PanelRow>
+					<DestructivedButton
+						onClick={ () => {
+							if ( window.confirm( __( 'Are you sure you want to cancel the test?', 'altis-experiments' ) ) ) {
+								updateTest( {
+									end_time: Date.now(),
+								}, false, true );
+							}
+						} }
+					>
+						{ __( 'Cancel test', 'altis-experiments' ) }
+					</DestructivedButton>
+				</PanelRow>
+			) }
+		</Fragment>
+	);
+};
+
+const SettingsWithData = compose(
+	withTestData,
+)( Settings );
```

Comment:
Unnecessary use of `compose`; can reduce to `export default withTestData( Settings )`